### PR TITLE
Feature #141 - Implement view for order creation

### LINKED ISSUE #141: SP: 8 Implement View for order creation
Part of user story #37

### PR DESCRIPTION
**Pull request by Misha86**

### CODE REVIEW

File: beauty/api/serializers/business_serializers.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+from api.models import Business, CustomUser
+
+
+class BusinessListCreateSerializer(serializers.ModelSerializer):
```
Comment (pazuzu-ua) [inline beauty/api/serializers/business_serializers.py:5]: Please, add a docstring

File: beauty/api/serializers/business_serializers.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    class Meta:
```
Comment (pazuzu-ua) [inline beauty/api/serializers/business_serializers.py:6]: Please, add a docstring

File: beauty/api/serializers/business_serializers.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        model = Business
+        fields = ('name', 'type', 'owner', 'description')
+
+    def to_representation(self, instance):
```
Comment (pazuzu-ua) [inline beauty/api/serializers/business_serializers.py:10]: Please, add a docstring

File: beauty/api/views.py [hunk 4]
```diff
@@ -89,4 +90,29 @@ def get_object(self):
                                 Q(specialist=self.kwargs['user']),
                                 id=self.kwargs['id'])
         self.check_object_permissions(self.request, obj)
-        return obj
\ No newline at end of file
+        return obj
+
+
+class BusinessListCreateView(ListCreateAPIView):
```
Comment (pazuzu-ua) [inline beauty/api/views.py]: Please, add a docstring

File: beauty/api/views.py [hunk 5]
```diff
@@ -89,4 +90,29 @@ def get_object(self):
...
+    queryset = Business.objects.all()
+    serializer_class = BusinessListCreateSerializer
+
+    def get_permissions(self):
```
Comment (pazuzu-ua) [inline beauty/api/views.py]: Please, add a docstring

**Review by pazuzu-ua — COMMENTED** [review]
Good one! Just add some docstrings...

File: beauty/api/views.py (hunk 4 above)
Comment (Misha86) [reply beauty/api/views.py]: Business is not my task.

File: beauty/api/serializers/business_serializers.py (hunk 3 above)
Comment (Misha86) [reply beauty/api/serializers/business_serializers.py:10]: Business is not my task.

File: beauty/api/serializers/business_serializers.py (hunk 1 above)
Comment (Misha86) [reply beauty/api/serializers/business_serializers.py:5]: Business is not my task.

File: beauty/api/views.py (hunk 5 above)
Comment (Misha86) [reply beauty/api/views.py]: Business is not my task.

File: beauty/api/serializers/business_serializers.py (hunk 1 above)
Comment (Deer-WarLord) [reply beauty/api/serializers/business_serializers.py:5]: add TODO doc + responsible person

File: beauty/api/serializers/order_serializers.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+from api.models import (Order, CustomUser, Service, Position)
+
+
+class SpecialistRelatedField(serializers.PrimaryKeyRelatedField):
+    """Custom PrimaryKeyRelatedField for specialist users."""
+
+    def get_queryset(self):
+        """Get all specialists."""
+        return self.queryset.filter(
+            groups__name__icontains="specialist")
```
Comment (Deer-WarLord) [inline beauty/api/serializers/order_serializers.py:13]: Should it be `contains` or strong equal?

**Pull request closed without merging**